### PR TITLE
Fission/update sector nutrition

### PR DESCRIPTION
**Pull request by sayyedhanif1**

### CODE REVIEW

File: dimagi/pages/static/style/pages/sectors/nutrition_programs.scss [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  @include make-bg(hero-data-image, '../../../pages/sectors/nutrition/intro/hero', $_base-height, $_height-by-size);
+}
+
+.news-images .image.blog1 {
+  @include background-cover('../../../pages/sectors/nutrition/news/Blog1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.news-images .image.blog2 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/nutrition_programs.scss]: new line before style definition

File: dimagi/pages/static/style/pages/sectors/nutrition_programs.scss [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  @include background-cover('../../../pages/sectors/nutrition/news/Blog2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.news-images .image.blog3 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/nutrition_programs.scss]: add new line before style definition

File: dimagi/pages/static/style/pages/sectors/nutrition_programs.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  @include background-cover('../../../pages/sectors/nutrition/news/Blog3.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.programs-images .image.featured-project1 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/nutrition_programs.scss]: add new line before style definition

File: dimagi/pages/static/style/pages/sectors/nutrition_programs.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  @include background-cover('../../../pages/sectors/nutrition/programs/FeaturedProject1.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.programs-images .image.featured-project2 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/nutrition_programs.scss]: add new line before style definition

File: dimagi/pages/static/style/pages/sectors/nutrition_programs.scss [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  @include background-cover('../../../pages/sectors/nutrition/programs/FeaturedProject2.jpg');
+  height: 200px;
+  background-size: contain;
+  background-position: left;
+}
+.programs-images .image.featured-project3 {
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/nutrition_programs.scss]: add new line before style definition

File: dimagi/pages/templates/pages/sectors/nutrition_programs.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  <!--[if lte IE 8]>
+    <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
+  <![endif]-->
+  <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+  <script>
+    //  Schedule demo form
+    hbspt.forms.create({
+      portalId: "503070",
+      formId: "afb4d5a0-826f-4a56-8c21-552d4b6dafa3",
+      target: "#schedule-demo-form-content",
```
Comment (biyeun) [inline dimagi/pages/templates/pages/sectors/nutrition_programs.html]: why is `demoForm.js` not being used here?

Comment (sayyedhanif1) [reply dimagi/pages/templates/pages/sectors/nutrition_programs.html]: Initially, I took this code from existing code. Here also demoForm.js have diff. formId ( "fafedecc-c3d8-4e10-a3fe-bd8978162da1" ) and here is "afb4d5a0-826f-4a56-8c21-552d4b6dafa3" .
That's why also I didn't used this.
So here we need to add new js file for loading this form or else used only demoForm.js?

File: dimagi/pages/static/style/pages/sectors/nutrition_programs.scss [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+    small: 280px,
+    medium: 280px,
+    large: 280px,
+    extra-large: 340px,
+  );
+  @include make-bg(hero-data-image, '../../../pages/sectors/nutrition/intro/hero', $_base-height, $_height-by-size);
+}
+
+.news-images .image.blog1 {
+  @include background-cover('../../../pages/sectors/nutrition/news/Blog1.jpg');
```
Comment (biyeun) [inline dimagi/pages/static/style/pages/sectors/nutrition_programs.scss]: please see the file naming conventions comment i made in https://github.com/dimagi/dimagi.com/pull/92

File: dimagi/pages/templates/sections/sectors/nutrition_programs/part3.html [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+            {% endblocktrans %}
+          </p>
+          <p class="cta">
+            {% trans 'Watch Now' as video_text %}
+            {% click_hero 'video' as video_click_class %}
+            {% include "partials/wistia_popover.html" with wistia_id='jzm3ggrinr' show_play=True text=video_text click_class=video_click_class %}
+          </p>
+        </div>
+      </div> {# .content #}
+      <div class="phone-image off-grid white-lazy-bg"
```
Comment (biyeun) [inline dimagi/pages/templates/sections/sectors/nutrition_programs/part3.html]: please see the comment I made about the phone image in https://github.com/dimagi/dimagi.com/pull/92

File: dimagi/pages/static/style/pages/sectors/nutrition_programs.scss (hunk 7 above)
Comment (biyeun) [reply dimagi/pages/static/style/pages/sectors/nutrition_programs.scss]: no capitalization in the image filenames and no CamelCase

**Pull request merged**